common/arch/sdl/event.cpp

```diff
@@ -44,6 +49,29 @@ static void windowevent_handler(const SDL_WindowEvent &windowevent)
 }
 #endif
 
+static void syswmevent_handler(SDL_SysWMmsg *message)
+{
+#if defined(_WIN32) && SDL_MAJOR_VERSION == 2
+#define MM_MCINOTIFY 0x3B9
+	if (message->subsystem == SDL_SYSWM_WINDOWS) {
+		con_printf(CON_VERBOSE, "SYSWM message M:%08x (%6d) - W:%08x L:%08lx", message->msg.win.msg, message->msg.win.msg, message->msg.win.wParam, message->msg.win.lParam);
+		if (message->msg.win.msg == MM_MCINOTIFY)
+			RBAGotMCIEvent(message->msg.win.wParam, message->msg.win.lParam);
+	}
+#else
+	message
```

Comment:
This should be `(void)message;` if you want to suppress the unused variable diagnostic.